src/runtime_src/tools/scripts/apu_recipes/apu-boot

```diff
@@ -5,4 +5,8 @@ if [ -e /sys/bus/platform/devices/rpu-channel/ready ]; then
 	useradd softkernel
 	# Work-around for AIE as it defaults root access only
 	chmod 666 /dev/aie0
+	# Create default xrt.ini
+	echo "[Runtime]" > /usr/bin/xrt.ini
```

Comment:
Add a comment to the xrt.ini that this was generated by APU installation script.